src/go/k8s/pkg/resources/statefulset_update.go

```diff
@@ -214,6 +214,25 @@ func (r *StatefulSetResource) queryRedpandaForTopicMembers(
 	return consumer.Close()
 }
 
+func (r *StatefulSetResource) getCertSecret(
```

Comment:
should this come with its own test?

Reply:
we test only public API, this is private method. But perhaps this could be tested as part of some other public API, let me think about it.